<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1496</width>
    <height>722</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>12</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="ScenarioTacticalObjects" name="wTacticalSigns" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mapFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="mapLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbOpen">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resource.qrc">
             <normaloff>:/icons/images/open.png</normaloff>:/icons/images/open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbPlay">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resource.qrc">
             <normaloff>:/icons/images/play.png</normaloff>:/icons/images/play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbPause">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resource.qrc">
             <normaloff>:/icons/images/pause.png</normaloff>:/icons/images/pause.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbResume">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resource.qrc">
             <normaloff>:/icons/images/resume.png</normaloff>:/icons/images/resume.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbStop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resource.qrc">
             <normaloff>:/icons/images/stop.png</normaloff>:/icons/images/stop.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbReload">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resource.qrc">
             <normaloff>:/icons/images/reload.png</normaloff>:/icons/images/reload.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb3D">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resource.qrc">
             <normaloff>:/icons/images/3d.png</normaloff>:/icons/images/3d.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbSpeed">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>36</height>
            </size>
           </property>
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1496</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="mainMenu">
    <property name="title">
     <string>Сценарий</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveScenario"/>
    <addaction name="actionSaveAsScenario"/>
    <addaction name="separator"/>
    <addaction name="actionUpdateObjectsFromBD"/>
    <addaction name="actionSetMapCenter"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menuForceEdit">
    <property name="title">
     <string>Редактор сил и средств</string>
    </property>
    <addaction name="actionShowForceEdit"/>
    <addaction name="action_diagram"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="mainMenu"/>
   <addaction name="menuForceEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuOpenScenario">
   <property name="text">
    <string>Открыть сценарий</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSaveScenario">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Изменить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionShowForceEdit">
   <property name="text">
    <string>Показать</string>
   </property>
  </action>
  <action name="actionSaveAsScenario">
   <property name="text">
    <string>Сохранить как ..</string>
   </property>
  </action>
  <action name="actionUpdateObjectsFromBD">
   <property name="text">
    <string>Обновить объекты из БД</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actionSetMapCenter">
   <property name="text">
    <string>Распределить от центра </string>
   </property>
  </action>
  <action name="action_diagram">
   <property name="text">
    <string>Редактор поведенческого ядра объекта</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScenarioTacticalObjects</class>
   <extends>QWidget</extends>
   <header location="global">../../../db_service/ScenarioTacticalObjects.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
